ot/gromov/_utils.py

```diff
@@ -1,19 +1,21 @@
 # -*- coding: utf-8 -*-
 """
-Gromov-Wasserstein and Fused-Gromov-Wasserstein utils.
+Gromov-Wasserstein, Fused-Gromov-Wasserstein and unbalanced Gromov-Wasserstein utils.
 """
 
 # Author: Erwan Vautier <erwan.vautier@gmail.com>
 #         Nicolas Courty <ncourty@irisa.fr>
 #         Rémi Flamary <remi.flamary@unice.fr>
 #         Titouan Vayer <titouan.vayer@irisa.fr>
 #         Cédric Vincent-Cuaz <cedvincentcuaz@gmail.com>
+#         Huy Tran <huytran82125@gmail.com>
 #
 # License: MIT License
 
 
 from ..utils import list_to_array
 from ..backend import get_backend
+from ot.unbalanced import sinkhorn_unbalanced, mm_unbalanced, lbfgsb_unbalanced
```

Comment:
update `ot.solve` to support latest new features for unbalanced solvers such that new wrapper `uot_solver` becomes unnecessary.